app/src/main/java/fr/free/nrw/commons/nearby/mvp/fragments/NearbyParentFragment.java

```diff
@@ -0,0 +1,884 @@
+            wikipediaButtonText.setVisibility(View.GONE);
+            wikidataButtonText.setVisibility(View.GONE);
+            commonsButtonText.setVisibility(View.GONE);
+            directionsButtonText.setVisibility(View.GONE);
+        }
+    }
+
+    /**
+     *
+     */
+    private void addActionToTitle() {
+        title.setOnClickListener(view -> {
+            Utils.copy("place", title.getText().toString(), getContext());
+            Toast.makeText(getContext(), "Text copied to clipboard", Toast.LENGTH_SHORT).show();
+            if (bottomSheetDetailsBehavior.getState() == BottomSheetBehavior.STATE_COLLAPSED) {
+                bottomSheetDetailsBehavior.setState(BottomSheetBehavior.STATE_EXPANDED);
+            } else {
+                bottomSheetDetailsBehavior.setState(BottomSheetBehavior.STATE_COLLAPSED);
+            }
+        });
+    }
+
+    /**
+     * Returns the list fragment added to child fragment manager previously, if exists.
+     */
+    private NearbyListFragment getListFragment() {
+        NearbyListFragment existingFragment = (NearbyListFragment) getChildFragmentManager()
+                .findFragmentByTag(TAG_RETAINED_LIST_FRAGMENT);
+        if (existingFragment == null) {
+            existingFragment = setListFragment();
+        }
+        return existingFragment;
+    }
+
+    /**
+     * Returns the map fragment added to child fragment manager previously, if exists.
+     */
+    private NearbyMapFragment getNearbyMapFragment() {
+        NearbyMapFragment existingFragment = (NearbyMapFragment) getChildFragmentManager()
+                .findFragmentByTag(TAG_RETAINED_MAP_FRAGMENT);
+        if (existingFragment == null) {
+            existingFragment = setMapFragment();
+        }
+        return existingFragment;
+    }
+
+    /**
+     * Creates the map fragment and prepares map
+     * @return the map fragment created
+     */
+    private NearbyMapFragment setMapFragment() {
+        // Mapbox access token is configured here. This needs to be called either in your application
+        // object or in the same activity which contains the mapview.
+        Mapbox.getInstance(getActivity(), getString(R.string.mapbox_commons_app_token));
+        NearbyMapFragment mapFragment;
+
+        // Create fragment
+        FragmentTransaction transaction = getChildFragmentManager().beginTransaction();
+
+        // Build mapboxMap
+        isDarkTheme = applicationKvStore.getBoolean("theme", false);
+        MapboxMapOptions options = new MapboxMapOptions()
+                .compassGravity(Gravity.BOTTOM | Gravity.LEFT)
+                .compassMargins(new int[]{12, 0, 0, 24})
+                .logoEnabled(false)
+                .attributionEnabled(false)
+                .camera(new CameraPosition.Builder()
+                        .zoom(ZOOM_LEVEL)
+                        .target(new com.mapbox.mapboxsdk.geometry.LatLng(-52.6885, -70.1395))
```

Comment:
Which coordinate is this? Extract to a named constant if it is just a default value.